package/rmt-server.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Fri Jun 15 13:12:09 UTC 2018 - ikapelyukhin@suse.com
+
+- Log version on startup
```

Comment:
Is it maybe worth mentioning that it does this only when ran as a service?
Now you could think it would also log it when started manually on the command line.

Reply:
OK!